### PR TITLE
Factor out header/footer and other minor fixes

### PR DESCRIPTION
**Pull request by margaretchan**

### CODE REVIEW

**Comment by margaretchan** [comment]
Primary Reviewer: Derrill
Secondary Reviewer: Sam

File: portfolio/src/main/webapp/projects.html [hunk 1]
```diff
@@ -7,19 +7,7 @@
         <script src="script.js"></script>
```
Comment (derrillhd) [inline portfolio/src/main/webapp/projects.html:7]: You should add a comment saying the header and footer is rendered in script.js by it setting the document.querySelector.

It's not obvious by looking at it from here.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -7,19 +7,7 @@
...
-                <li><a href="about.html">ABOUT</a></li>
-                <li><a href="projects.html">PROJECTS</a></li>
-                <li><a href="campus.html">CAMPUS INVOLVEMENT</a></li>
-                <li><a href="blog.html">BLOG</a></li>
-            </ul>
-        </nav>
-    </header>
-
-    <hr>
+    <header></header>
```
Comment (derrillhd) [inline portfolio/src/main/webapp/index.html:10]: You should add a comment saying that body of this is set to header.html in script.js which sets innerHTML via  document.querySelector.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -34,12 +22,6 @@ <h2>Cornell University '22</h2>
         </div>
     </body>
 
-    <hr>
+    <footer></footer>
```
Comment (derrillhd) [inline portfolio/src/main/webapp/index.html:25]: You should add a comment saying that body of this is set to footer.html in script.js which sets innerHTML via  document.querySelector.

**Pull request merged**